application/models.py

```diff
@@ -274,39 +292,14 @@ class LandingPages(Page):
     description_sv = models.TextField(null=True, blank=True, verbose_name='Selite SV')
     description_en = models.TextField(null=True, blank=True, verbose_name='Selite EN')
 
-    title_and_description_color_fi = models.CharField(max_length=255, choices=title_and_description_color_choices, null=True, blank=True, verbose_name='Tekstin väri FI')
```

Comment:
May I ask the reasoning behind removing `title_and_description_color` set of fields from `BannerPages`? To me, the banner includes the `title` and `description` and having the `title_and_description_color` which would set the color of both title and description seems inherently part of a Banner.

Reply:
Fixed. That's my mistake, for some reason those fields was left out and I didn't notice that 😥 That's why there is an unused `title_and_description_color_choices` above